service/controller/app/v1/resource/chart/create_test.go

```diff
@@ -109,15 +82,15 @@ func Test_Resource_newCreateChange(t *testing.T) {
 		t.Fatalf("error == %#v, want nil", err)
 	}
 
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got, err := r.newCreateChange(context.Background(), tt.currentResource, tt.desiredResource)
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			result, err := r.newCreateChange(context.Background(), tc.currentResource, tc.desiredResource)
 			if err != nil {
-				t.Errorf("Resource.newCreateChange() error = %v", err)
-				return
+				t.Fatalf("error == %#v, want nil", err)
 			}
-			if !reflect.DeepEqual(got, tt.expectedChart) {
-				t.Errorf("Resource.newCreateChange() = %v, want %v", got, tt.expectedChart)
+
+			if !reflect.DeepEqual(result, tc.expectedChart) {
+				t.Fatalf("want matching chart \n %s", cmp.Diff(result, tc.expectedChart))
```

Comment:
Aligning with configmap and secret resources and making the diffs easier to read.